classpath_validator.bzl

```diff
@@ -24,15 +24,20 @@ def _impl(ctx):
         content = "\n".join([(_label_as_string(j.owner) + " " + j.short_path) for j in runtime_jars]),
     )
 
-    ignore_prefixes_file = _write_ignore_file(ctx, ctx.attr.ignore_prefixes, ctx.label.name + "_ignore_prefixes.txt")
-    ignore_suffixes_file = _write_ignore_file(ctx, ctx.attr.ignore_suffixes, ctx.label.name + "_ignore_suffixes.txt")
+    ignore_prefixes_file = _write_entries_file(ctx, ctx.attr.ignore_prefixes, ctx.label.name + "_ignore_prefixes.txt")
```

Comment:
I wonder if there's a way to write a single file with all the entries instead of 4 files.
The reason I'm asking is because each `ctx.actions.write` generates a new action (RBE worker / Bazel worker).

I know that in my original implementation I already had two writes but now we doubled it.

Note that we can serialize a struct to json and then read it from Java but I'd like to keep the Java implementation free of external dependencies (like jackson).

Reply:
We could pass the prefixes and suffixes as command line arguments, e.g. 
```
---include_prefixes prefix1 prefix2 --include_suffixes suffix1 suffix2
```

but it would require a custom argument parser on Java's side

Reply:
What if we create a single input file that would look like this:
```properties
include.prefix=pref1,pref2
include.suffix=suff1,suff2
exclude.prefix=pref3
exclude.suffix=suff3
```
And this would be a standard Java Properties file

Reply:
I would be -1 one this one - eventually you have to start worrying about escaping control characters, and other fun things to be compatible with a spec / standard, and startlark just isn't a language suited for that (it doesn't even have support for regex)